### ISSUE TITLE
Animate Height As well

### ISSUE THREAD

**Issue by johnnyboyuk84** [jmar777/kwicks#11]
Hi, great plugin btw. 

I'd like to animate the height of the li as well as the width on a rollover. Thus allowing for a bigger menu items to be shown. Then close back up saving space. 

I've looked at the code and I'm unsure where to edit the code to add this in. Do you have an idea where i should start as its not obvious to a novice.

Thanks
John

**Comment by jmar777** [comment]
Hey, sorry for the ridiculously long delay in replying.  I realize this is probably too late to be helpful at this point, but are you still having any issues I can help with?

**Comment by johnnyboyuk84** [comment]
Haha thank you for your reply anyway, but I did manage to solve the issue with a bit of jquery and CSS, made a container around the kwick slider and animate the height of the container on mouse over and hey presto. The only problem was at times it seemed to be  a little bit slow working along side your JavaScript and compatibility with older browser was not great so I decided to remove it and go with a simpler mega menu instead.

Maybe something to add in your plugin in future updates though as it did look good.

Thanks for your reply, much appreciated :)

---

John Emmett
East lancs web design
07816097516
